Clarify phrasing on configSections deep-dive-into-iis-configuration-with-iis-7-and-iis-8.md (#929)
MicrosoftDocs/iis-docs: iis/get-started/planning-your-iis-architecture/deep-dive-into-iis-configuration-with-iis-7-and-iis-8.md

> [!NOTE]
> Customers and third parties should not attempt to change schema information for the built-in sections, either in the inetsrv\config\schema\ folder or in machine.config and applicationHost.config. This may yield to undesirable behavior for these sections.

> The content of `<configSections>` is a list of sections that are "registered" with the system (this is their registration point). It also defines the hierarchy of section groups. It does not, however, defines properties (or elements) within sections. Some additional metadata is defined for sections:

Reviewer: ```suggestion
The content of `<configSections>`:
* Is a list of sections that are *registered* with the system.
* Provide the registration point.
* Defines the hierarchy of section groups.
* Does ***not** define properties or elements within sections.

Some additional metadata is defined for sections:
```

Reviewer 2: Coincidentally, I noticed this issue in the docs just today. My thoughts:

The first bullet point doesn't make it clear that this is the section that _defines_ the config sections. That's what the `(this is the registration point)` was trying to clarify. The second bullet point isn't very meaningful or clear on its own.  

Since this is an old PR and I'm not sure the author is still active, I made my own PR with my proposed changes: https://github.com/MicrosoftDocs/iis-docs/pull/1048
